src/API/Site/Controllers/FeaturesController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\TransportMethods;
+use Automattic\WooCommerce\GoogleListingsAndAds\Options\OptionsInterface;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\RESTServer;
+use WP_REST_Request as Request;
+use Exception;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class FeaturesController
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers
+ */
+class FeaturesController extends BaseOptionsController {
+
+	/**
+	 * Constructor.
+	 *
+	 * @param RESTServer $server
+	 */
+	public function __construct( RESTServer $server ) {
+		parent::__construct( $server );
+	}
+
+	/**
+	 * Register rest routes with WordPress.
+	 */
+	public function register_routes(): void {
+		/**
+		 * GET The tour visualizations
+		 */
+		$this->register_route(
+			'/features',
+			[
+				[
+					'methods'             => TransportMethods::READABLE,
+					'callback'            => $this->get_features_read_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+					'args'                => $this->get_schema_properties(),
+				],
+				'schema' => $this->get_api_response_schema_callback(),
+			],
+		);
+	}
+
+	/**
+	 * Callback function for returning the tours
+	 *
+	 * @return callable
+	 */
+	protected function get_features_read_callback(): callable {
+		return function () {
+			return $this->options->get( OptionsInterface::WCS_FEATURE_FLAGS, [] );
```

Comment:
Rather than checking an option, I'd suggest implementing a getter on the Features service so you can decouple the storage mechanism from the data retrieval. Something like:

```
return $this->features->get_features();
```

A nice to have addition to this is that if the endpoint supported passing a specific feature flag, you could call `$this->features->get_features( [$feature] );` to return an object for only that one feature. You just need to ensure that the processing that is currently happening in `Features:update()` is still being applied to either after the WCS request is returned, or as part of the getter before the raw feature flag values are returned here.